tailwind.config.js

```diff
@@ -6,6 +6,13 @@ module.exports = {
     './src/**/*.tsx',
   ],
   theme: {
+    screens: {
```

Comment:
tailwind lets you set this arbitrarily, but 1900 felt pretty good on my 32" screen